.travis.yml

```diff
@@ -1,22 +1,23 @@
 language: php
 
 php:
-  - 5.6
-  - 7.0
-  - 7.1
+    - 5.6
```

Comment:
IMO we should drop support for old PHP versions now that are out of support.
They can always install the old tagged version anyway.

(same for old laravel versions not in support)

Reply:
I agree, I left it in because I didn't want to make that call but I'm sure it certainly makes it easier to maintain.  That might fit in a separate PR.